openslides_backend/action/actions/user/user_mixin.py

```diff
@@ -19,4 +24,158 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 raise ActionException(
                     f"A user with the username {instance['username']} already exists."
                 )
+        self.check_existence_of_to_and_from_users(instance)
+        self.check_meeting_and_users(instance, user_fqid)
+        if "vote_delegated_$_to_id" in instance:
+            self.check_vote_delegated__to_id(instance, user_fqid)
+        if "vote_delegations_$_from_ids" in instance:
+            self.check_vote_delegations__from_ids(instance, user_fqid)
         return instance
+
+    def check_vote_delegated__to_id(
+        self, instance: Dict[str, Any], user_fqid: FullQualifiedId
+    ) -> None:
+        mapped_fields = [
+            f"vote_delegations_${meeting_id}_from_ids"
+            for meeting_id, delegated_to in instance["vote_delegated_$_to_id"].items()
+            if delegated_to
+        ]
+        if not mapped_fields:
+            return
+        user_self = self.datastore.fetch_model(
+            user_fqid, mapped_fields, lock_result=True, exception=False
+        )
+        if "vote_delegations_$_from_ids" in instance:
+            update_dict = {
+                f"vote_delegations_${meeting_id}_from_ids": delegated_from
+                for meeting_id, delegated_from in instance[
+                    "vote_delegations_$_from_ids"
+                ].items()
+            }
+            user_self.update(update_dict)
```

Comment:
I think this would override any given `vote_delegated__to_id` from the same meeting, right?

Reply:
Please add a test where you give e.g. `vote_delegated_$1_to_id` as well as `vote_delegated_$1_from_ids` and check that this fails (if you didn't already do that, I did not find it among the tests).

Reply:
> I think this would override any given `vote_delegated__to_id` from the same meeting, right?
Yes, if it is changed it should overwrite it

Reply:
> Please add a test where you give e.g. `vote_delegated_$1_to_id` as well as `vote_delegated_$1_from_ids` and check that this fails (if you didn't already do that, I did not find it among the tests).

There have been 3 tests, but I added one more test, checking for error. IMO it works quite well

Reply:
ok